### ISSUE TITLE
Dup detection

### ISSUE THREAD

**Issue by wumpus** [skandragon/thing#102]
I got scheduled for 2 classes on Sat 8/2 noon. Thing does appear to understand that I'm a single person, and I realize that the dance class track is scheduled separately, and there was probably a human who was supposed to notice this problem, but it might be nice(tm) if software provided a double-check.

**Comment by maco** [comment]
As a point of clarification: you were scheduled to *teach* two classes simultaneously, right?

**Comment by Artimisia** [comment]
This is Artis (registrar) - I am looking at your classes, Greg and not seeing the issue (or maybe I fixed it already, and don’t recall doing it-which is possible in the rush).  Your classes are:
	4-15th Cen. Italian Dances						7/29 @ 4pm in Dance Tent
	History of Dance								7/30 @ noon in Dance Tent
	Elizabethan Era Folk Tunes						7/30 @ 4pm in the PART
	Steps of 15th-Cen. Italian Dancing					7/31 @ 10 in the Dance Tent
	Longways										8/01 @ 1pm in the Dance Tent
	Elizabethan Era Folk Tunes 						8/02 @ 3pm in the PART
	Digitizing and translating 16th-Cen. Dance			8/06 @ 10am in the Dance Tent

I suspect we may have found the conflict when we ran the conflict checking reports, and corrected it - while you had looked between when the classes were scheduled and the checking done.  
I apologize for any problems this has caused.
-Artis

**Comment by maco** [comment]
The original issue was posted for Pennsic 43

**Comment by wumpus** [comment]
Yes, this issue was for Pennsic 43. (Sorry I didn't reply earlier, github decided to stop emailing me.)
